nemo/utils/exp_manager.py

```diff
@@ -119,6 +121,9 @@ class ExpManagerConfig:
     # Configures creation of log files for different ranks
     log_local_rank_0_only: Optional[bool] = False
     log_global_rank_0_only: Optional[bool] = False
+    # todo (sean): might not be the best place to add this, but allows us to enable it fast
```

Comment:
Maybe we can put this as part of our optimizer's configs?

Reply:
Not part of optim/sched. Those pertain only the the constructors of the optimizer and scheduler, not logging.

Reply:
It's a bit tricky to feel where exactly this should go. @titu1994 is there a registry of callbacks somewhere that may be this can be registered to add to the callbacks list?

Reply:
Exp manager technically has no registry, but that's a very good idea to implement. We already have the timer callback that we inject into ExpManager maybe lets register that as well as this one